app/views/machines/show.html.erb

```diff
@@ -1 +1,6 @@
 <h1><%= @machine.location %> Vending Machine</h1>
+<% @machine.snacks.each do |snack| %>
+  <p><%=snack.name%>: $<%=snack.price%></p>
```

Comment:
so you could use the action view helper here like number_to_currenct(snack.price)